ilastik/workflows/tracking/structured/structuredTrackingWorkflow.py

```diff
@@ -109,17 +113,26 @@ def __init__( self, shell, headless, workflow_cmdline_args, project_creation_arg
         self.default_tracking_export_filename = '{dataset_dir}/{nickname}-tracking_exported_data.csv'
         self.dataExportTrackingApplet = TrackingBaseDataExportApplet(self, "Tracking Result Export",default_export_filename=self.default_tracking_export_filename)
         opDataExportTracking = self.dataExportTrackingApplet.topLevelOperator
-        opDataExportTracking.SelectionNames.setValue( ['Tracking Result', 'Merger Result', 'Object Identities'] )
+        opDataExportTracking.SelectionNames.setValue( ['Tracking-Result', 'Merger-Result', 'Object-Identities'] )
         opDataExportTracking.WorkingDirectory.connect( opDataSelection.WorkingDirectory )
```

Comment:
Thanks for the hint. We had hyphens in the Conservation Tracking workflow since @JaimeIvanCervantes changed it some time ago (https://github.com/ilastik/ilastik/commit/6e3a8be8c34115cfcce746d29f989848fc58585c). 
@ilastikdev, could you please adjust that for both workflows? Thanks